docs/spaces/index.asciidoc

```diff
@@ -2,87 +2,77 @@
-After you create your own
-spaces, you're asked to choose a space when you log in to {kib}. You can change your
-current space at any time by using the menu.
+When you create more spaces, users are asked to choose a space when they log in to {kib}, and can change their
+current space at any time from the top menu.
 
 [role="screenshot"]
 image::images/change-space.png["Change current space menu"]
 
+To go to **Spaces**, find **Stack Management** in the navigation menu or use the <<kibana-navigation-search,global search bar>>.
+
 [float]
-==== Required privileges
+=== Required privileges
 
 The `kibana_admin` role or equivalent is required to manage **Spaces**.
 
 [float]
 [[spaces-managing]]
-=== View, create, and delete spaces
-
-Open the main menu, then click *Stack Management > Spaces* for an overview of your spaces.  This view provides actions
-for you to create, edit, and delete spaces.
+=== Create a space
 
-[role="screenshot"]
-image::images/space-management.png["Space management"]
-
-[float]
-==== Create or edit a space
-
-You can create as many spaces as you like. Click *Create a space* and provide a name,
-URL identifier, optional description.
+You can create as many spaces as you like. 
 
+. Select *Create space* and provide a name, description, and URL identifier.
++
 The URL identifier is a short text string that becomes part of the
 {kib} URL when you are inside that space. {kib} suggests a URL identifier based
 on the name of your space, but you can customize the identifier to your liking.
 You cannot change the space identifier once you create the space.
 
-{kib} also has an <<spaces-api, API>>
-if you prefer to create spaces programmatically.
-
-[role="screenshot"]
-image::images/edit-space.png["Space management"]
+. Select a **Solution view**. This setting controls the navigation that all users of the space will get:
 
-[float]
-==== Delete a space
-
-Deleting a space permanently removes the space and all of its contents.
-Find the space on the *Spaces* overview page and click the trash icon in the Actions column.
-You can't delete the default space, but you can customize it to your liking.
+** **Search**: A light navigation menu focused on analytics and Search use cases. Features specific to Observability and Security are hidden.
+** **Observability**: A light navigation menu focused on analytics and Observability use cases. Features specific to Search and Security are hidden.
+** **Security**: A light navigation menu focused on analytics and Security use cases. Features specific to Observability and Search are hidden.
+** **Classic**: All features from all solutions are visible by default using the classic, multilayered navigation menus. You can customize which features are visible individually. 
 
-[float]
 [[spaces-control-feature-visibility]]
-=== Control feature access based on user needs
+. If you selected the **Classic** solution view, you can customize the **Feature visibility** as you need it to be for that space.
```

Comment:
The numbered list is restarting here

Reply:
Indeed the ID discontinues the list, fixing

Reply:
Looks good!